package.json

```diff
@@ -52,14 +53,17 @@
     "scss-unused-variables": "./scripts/find_scss_unused_variables.sh src/",
     "build": "gulp build",
     "start": "http-server ./dist -p 8000",
-    "prepush": "npm test",
+    "prepush": "yarn test",
```

Comment:
why you change this? There is no difference.
But if you wanna change do it also on line 60, task test

Reply:
why mix yarn and npm ?:D
I made changes as you requested

Reply:
everywhere yarn is ok 👍